### PR TITLE
Remove all overwriting of objects with same name

### PR DESCRIPTION
**Pull request by KristinaRiemer**

### CODE REVIEW

File: vignettes/04-synthesis-data.Rmd [hunk 1]
```diff
@@ -31,7 +31,7 @@ trait_canopy_height <- betydb_query(table     = "search",
                       trait     = "canopy_height", 
                       date      = "~2017",
                       limit     =  "none")
-trait_canopy_height = trait_canopy_height %>% 
+trait_canopy_height2 = trait_canopy_height %>%
```
Comment (Chris-Schnaufer) [inline vignettes/04-synthesis-data.Rmd]: I don't like names that are too long, but maybe trait_canopy_height_day or trait_canopy_height_by_day?

File: vignettes/04-synthesis-data.Rmd [hunk 2]
```diff
@@ -240,7 +236,7 @@ trait_canopy_cover <- betydb_query(table     = "search",
                        date      = "~2018 May",
                        limit     =  "none")
                        
-trait_canopy_cover <- trait_canopy_cover %>% 
+trait_canopy_cover2 <- trait_canopy_cover %>%
```
Comment (Chris-Schnaufer) [inline vignettes/04-synthesis-data.Rmd]: Same as above; maybe trait_canopy_cover_by_day?

File: vignettes/04-synthesis-data.Rmd [hunk 3]
```diff
@@ -250,12 +246,12 @@ We then determine the average canopy cover across the site for the day that the
 The relationship between our greenness metric and average canopy cover are plotted. 
 
 ```{r plot_sensor_trait}
-trait_canopy_cover <- trait_canopy_cover %>% 
+trait_canopy_cover3 <- trait_canopy_cover2 %>%
```
Comment (Chris-Schnaufer) [inline vignettes/04-synthesis-data.Rmd]: Maybe use traits_canopy_cover_and_greeness or traits_merged? I think it'd be clearer

Comment (dlebauer) [reply vignettes/04-synthesis-data.Rmd]: daily_canopy_cover?

**Comment by dlebauer** [comment]
Can you please make a new issue to update the models once the new data is generated?

**Pull request merged**